CMakeLists.txt

```diff
@@ -38,7 +38,11 @@ set(ROCM_PATH /opt/rocm CACHE PATH "Default ROCm installation path")
 if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
   set(CMAKE_INSTALL_PREFIX ${ROCM_PATH} CACHE PATH "MIVisionX default installation path" FORCE)
 endif(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+
+# Changes for SWDEV-310152: 
+# Since all public interface libraries are present in same folder 
+# RPATH/RUNPATH is not required
+set(CMAKE_SKIP_RPATH TRUE)
```

Comment:
Using RPATH should not be a problem to build and install libs. Is there are reason to disable it?